packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,22 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+- Add extra information to the label with `span.pt-label-description`.
+
+- Putting the `<input>` element _inside_ a `<label>` element increases the area where the user
+can click to activate the control. Notice how in the examples below, clicking a `<label>` focuses its `<input>`.
```

Comment:
If that's the way we're going, how about also explicitly putting that this **will** be deprecated in 2.0?